pypi_to_import

```diff
@@ -0,0 +1,5 @@
+six:six
```

Comment:
Can you make your tool assume that the two names match each other unless told otherwise? That'd keep these files shorter.

Reply:
i persume so. will look into it once ive got the base logic sound